src/GitVersion.Core/Configuration/ConfigurationConstants.cs

```diff
@@ -19,9 +19,9 @@ internal static class ConfigurationConstants
     public const string MainBranchRegex = "^master$|^main$";
     public const string DevelopBranchRegex = "^dev(elop)?(ment)?$";
     public const string ReleaseBranchRegex = "^releases?[/-]";
-    public const string FeatureBranchRegex = "^features?[/-]";
+    public const string FeatureBranchRegex = "^features?[/-](?<BranchName>.+)";
     public const string PullRequestBranchRegex = @"^(pull|pull\-requests|pr)[/-]";
     public const string HotfixBranchRegex = "^hotfix(es)?[/-]";
     public const string SupportBranchRegex = "^support[/-]";
-    public const string UnknownBranchRegex = ".*";
+    public const string UnknownBranchRegex = "(?<BranchName>.*)";
```

Comment:
Sure, but the original expression was `.*` so i wanted to stay consistent. AFAIK branch names can't be empty strings so it make no difference really.

Reply:
Fixed

Reply:
Of course it makes no different. That is the reason why it would make sense to specify it with `(?<BranchName>.+)` like you did it previously on other places. The reason why is semantical nature: It makes the circumstance for all readers clear that the branch name cannot be null or empty.